core/test_network/src/lib.rs

```diff
@@ -297,7 +297,7 @@ where
     B: BlockSynchronizerTrait<Sumeragi = S, World = W>,
 {
     fn drop(&mut self) {
-        iroha_logger::error!(
+        iroha_logger::warn!(
```

Comment:
Perhaps this is a `debug` message. We only need these data when something goes wrong.

Reply:
I think info message is enough - it is the same as in the next comment